include/SFML/System/Vector2.inl

```diff
@@ -53,6 +53,138 @@ y(static_cast<T>(vector.y))
 }
 
 
+////////////////////////////////////////////////////////////
+template <typename T>
+T Vector2<T>::length() const
+{
+    static_assert(std::is_floating_point_v<T>, "Vector2::length() is only supported for floating point types");
+
+    using std::sqrt; // allow ADL
+
+    return sqrt(lengthSq());
```

Comment:
No issues here, just commenting on how satisfying it is to see these helpers used to implement other helpers.

Reply:
Didn't know about this method! It might mean however that the `x*x + y*y` part can no longer be computed at compile time, but that's going toward micro-optimization territory.

I'll adjust this 🙂